libs/sysaudio/src/main.zig

```diff
@@ -27,8 +27,8 @@ pub const Format = enum {
     F32,
 };
 
-pub const DeviceDescriptor = struct {
-    mode: ?Mode = null,
+pub const DeviceConfig = struct {
```

Comment:
I propose the following naming:

* `sysaudio.DeviceConfig` -> `sysaudio.Device.Options`
* `sysaudio.DeviceDescriptor` -> `sysaudio.Device.Properties`

This aligns us with WebGPU terminology, which is nice.